module6/lessons/Week5/PreppingForInterviews.md

```diff
@@ -0,0 +1,102 @@
+* Clean up any visible messes in your background.
+* Make sure everything in your background reflects what you *want* the interviewer to see.
+* Use a zoom background!  If you're not sure of the company culture, or are unable to put yourself into a good environment for the interview, use a zoom background.
+
+## Arrive Early
+You never know what might come up on the day of the interview, so plan to arrive 10-15 minutes early to account for traffic/parking/check-in.  Even if you are interviewing remotely, you should click on the join link **EARLY**; you never know when zoom might need to update!
+
+## Bring Supplies
+If you are interviewing in person, make sure to bring:
+* Note-taking supplies
+* Fully charged computer
+* A few copies of your resume
+
+If you are interviewing remotely, make sure you have handy:
+* A copy of your resume
+* note-taking supplies
+* Projects that you want to highlight pulled up and ready to highlight!
+
+<section class='call-to-action' markdown='1'>
+
+During your Pair Interview Practice tomorrow, you will interview someone based on one of the job descriptions linked below.  You will interview *for* the other job description.  Your instructor will assign you into groups of interviewers and interviewees for each.
+
+Spend some time now preparing to interview:
+* Determine 2 technical and 2 behavioral questions to ask based on your job description
+* ✅Send your questions to your instructor for approval!
+
+With your remaining time, and before tomorrow, prepare to be an interviewee:
+* Follow the checklist above!
+
+<section class='instructor-notes' markdown='1'>
+Double check that these job descriptions are still active :) 
+</section>
+
+[Job Description A](https://www.builtincolorado.com/job/software-engineer-ii-mga-systems/167712)
+[Job Description B](https://www.builtincolorado.com/job/junior-software-engineer/168683)
+
+
+
+</section>
+
+
+# How to Talk about Turing
+
+You all have spent 9 months on this Turing journey, so *anything* you want to share about **your own experience** is great!  But, you might get some questions about Turing more generally - here are some key points that you can reference:
+
+* Turing School is an accredited, non-profit, fully-remote computer programming school, focused solely on helping students develop the technical and professional skills need for a  fulfilling career in technology.
+* All of Turing's programs are assessment-based; students continually prove their understanding through assessments.
+* Though not *always* on offer, [Launch](https://turing.edu/launch) is a program that teaches Web Development in C#/.NET with a focus on backend development.
+    * Our [curriculum](https://launch.turing.edu/) is open source and can be shared with anyone!
+    * Launch students work on a combination of individual and group projects throughout the program.
+    * The curriculum and projects are designed for career-starters to give a broad understanding of web-development.
```

Comment:
Two other things unique to Launch I think are worth including. 

The PD focus is unique: Productive/Unproductive struggle, asking questions, and giving/receiving feedback.

The project where they jumped onto another team and contributed I think is also worth mentioning as a great very real world example.